addons/account_edi/views/account_edi_document_views.xml

```diff
@@ -2,18 +2,34 @@
 <odoo>
     <data>
         <record id="view_tree_account_edi_document" model="ir.ui.view">
-            <field name="name">Account.edi.document.tree</field>
+            <field name="name">EDI Documents</field>
```

Comment:
Better to not let the user bother with account.edi.document as it will be confusing about which info is there and which info is on the account.move, while we have everything on the account.move itself.